### DISCUSSION TITLE
Plot brush - any chance of enabling Plot tip selections?

### DISCUSSION THREAD

**Discussion by mishatsvelik** [observablehq/plot#2174] [Q&A]
Hi @Fil - I've been playing with the excellent Plot Brush experiment. Is there any chance that it will support pointer click events too? I.e. https://observablehq.com/d/6f3b6e6206c3451b.

**Comment by Fil** [comment]
Sure we should account for this scenario in https://github.com/observablehq/plot/pull/1653; but let's not disperse the discussion.
